Uniformly continuous functions on proper closed intervals in the real numbers (#1773)
UniMath/SymmetryBookFormalization: src/real-numbers/inequality-real-numbers.lagda.md

    iff-leq-real-ℚ = iff-leq-lower-real-ℚ x y
```

### The canonical map from integers to the reals preserves and reflects inequality

```agda
module _
  {x y : ℤ}
  where

  abstract
    preserves-leq-real-ℤ : leq-ℤ x y → leq-ℝ (real-ℤ x) (real-ℤ y)
    preserves-leq-real-ℤ x≤y =
      preserves-leq-real-ℚ (preserves-leq-rational-ℤ x≤y)

    reflects-leq-real-ℤ : leq-ℝ (real-ℤ x) (real-ℤ y) → leq-ℤ x y
    reflects-leq-real-ℤ x≤y =
      reflects-leq-rational-ℤ x y (reflects-leq-real-ℚ x≤y)
```

> ### The canonical map from natural numbers to the reals preserves and reflects inequality

Reviewer: ```suggestion
### The canonical inclusion of the natural numbers into the reals preserves and reflects inequality
```

  leq-iff-not-in-lower-cut-ℝ = (not-in-lower-cut-leq-ℝ , leq-not-in-lower-cut-ℝ)
```

> ### If `q` is in the lower cut of `x`, `real-ℚ q ≤ x`

Reviewer: ```suggestion
### If `q` is in the lower cut of `x`, then `real-ℚ q ≤ x`
```
